pkg/visor/visor.go

```diff
@@ -91,11 +93,14 @@ type Visor struct {
 
 	pidMu sync.Mutex
 
-	rpcListener net.Listener
-	rpcDialers  []*RPCClientDialer
+	cliL net.Listener
+	hvE  map[cipher.PubKey]chan error
```

Comment:
What do you think of renaming it to something more clear?

Reply:
@nkryuchkov I have changed it to the following:
```golang
hvErrs map[cipher.PubKey]chan error // errors returned when the associated hypervisor ServeRPCClient returns.
```
And added a comment as a bonus.